README.md

```diff
@@ -108,101 +108,86 @@ fetch('<url>', {
 
 If there are no tokens available for the given issuer or the issuer doesn't support raw token redemptions, the returned promise rejects with an error. Otherwise, the token is attached in the Sec-Trust-Token request header.
 
-If that sites needs a redemption attestation to forward to other parties, it can redeem an issuer.example token and receive a Signed Redemption Record that can be used as a redemption attestation using the Fetch API:
+If that sites needs a redemption attestation to forward to other parties, it can redeem an issuer.example token and receive a Redemption Record that can be used as a redemption attestation using the Fetch API:
 
 
 
 ```
 fetch('<issuer>/.well-known/trust-token', {
   trustToken: {
-    type: 'srr-token-redemption',
+    type: 'rr-token-redemption',
     issuer: <issuer>,
     refreshPolicy: {none, refresh}
   }
 }).then(...)
 ```
 
 
-If there are no tokens available for the given issuer, the returned promise rejects with an error. Otherwise, it invokes the PrivacyPass redemption protocol against the issuer, with the token (potentially, if specified by an extension, along with associated redemption metadata) attached in the Sec-Trust-Token request header and the resulting Signed Redemption Record (SRR) being expected in the Sec-Trust-Token response header.
+If there are no tokens available for the given issuer, the returned promise rejects with an error. Otherwise, it invokes the PrivacyPass redemption protocol against the issuer, with the token (potentially, if specified by an extension, along with associated redemption metadata) attached in the Sec-Trust-Token request header and the resulting Redemption Record (RR) being expected in the Sec-Trust-Token response header.
 
-The structure of the Signed Redemption Record (SRR) is:
+The RR is HTTP-only and JavaScript is only able to access/send the RR via Trust Token Fetch APIs. It is also cached in new first-party storage accessible only by these APIs for subsequent visits to that first-party.
```

Comment:
can you add a line that the RR is just an arbitrary blob of bytes from the issuer?